template/funcs.go

```diff
@@ -1166,26 +1169,22 @@ func blacklisted(...string) (string, error) {
 	return "", errors.New("function is disabled")
 }
 
-// sandboxedPath returns a path that's been prefixed by the sandbox path,
-// if any. If a sandbox path was provided, it will return an error if the
-// path falls outside the sandbox.
-func sandboxedPath(sandbox, s string) (string, error) {
-	path, err := filepath.Abs(filepath.Join(sandbox, s))
-	if err != nil {
-		return "", err
-	}
+// pathInSandbox determines whether a provided path falls within the sandbox.
+// returns an error only if the file can't be evaluated (missing, invalid
+// symlink, etc.)
+func pathInSandbox(sandbox, path string) (bool, error) {
```

Comment:
Returning both a bool and an error is redundant here as you always return false with an error and true when there is no error. So when you check `!ok` above, you would always use that error and ignore the returned one whenever this returned false.

I'd say skip the bool and just use the standard `if err != nil {return err}` on the result.

Reply:
It was feeling semantically weird to return an error on a yes/no question. But I think you're right, let's do that.

Reply:
Done!

Reply:
Thanks. And yeah, I get the semantic itch for having it return a bool to a yes/no question-y function. Maybe with a different name or something it wouldn't feel wrong or if it could be reworked such that the error could be handled internally (just changing the bool result). But as is, I think simplifying it is better.